platform/commonUI/edit/test/actions/SaveAsActionSpec.js

```diff
@@ -156,6 +156,28 @@ define(
                 expect(SaveAsAction.appliesTo(actionContext)).toBe(false);
             });
 
+            it("uses the editor capability to save the object", function () {
+                mockEditorCapability.save.andReturn(new Promise(function () {}));
```

Comment:
I think every relevant dependency already has its promises mocked; the reason `waitsFor` was needed is the same as this:

https://github.com/nasa/openmct/commit/86fcf190660bfd0bbd2d965b031c05c09bfe43e6#diff-696b8ae07d16ac15f27ed229404497ea

Slightly unrelated but dropping it here since it's a small change that I could do as part of this PR: `SaveAsAction` gets a `CreationService` as dependency but it doesn't seem to do anything with it (not even pass it to something else). It's been like this since its inception https://github.com/nasa/openmct/commit/26e368f52dc071e17d7ed8dba79834bc53d8f6bd#diff-55181241cfa7aebea9080d84a4490c0b

Can the dependency be removed, or am I missing something?